deploy/openshift/templates/che-server-template.yaml

```diff
@@ -24,7 +24,7 @@ objects:
   metadata:
     name: che
   roleRef:
-    name: admin
+    name: edit
```

Comment:
maybe it makes describe in the documentation that CHE_INFRA_OPENSHIFT_PROJECT must be set to:
1. the same with Che Server namespace
2. empty value then oauth provider must be used
3. any value different from Che Server namespace then cluster admin roles should be bound to Che service account (not sure that we need this way of configuration)

Reply:
Currently Che service account does not have cluster wide privileges so it is a general documentation improvement not really related to this particular PR. But good point.